From 7d53a7a8a3749f8611a06a8e9d8fc47ee0edea3e Mon Sep 17 00:00:00 2001
From: Mingcong Bai <jeffbai@aosc.io>
Date: Wed, 4 Sep 2024 22:18:51 +0800
Subject: [PATCH 27/35] [r38452] vd_ffmpeg/ve_lavc: Use frame_num instead of
 frame_number

Removed from lavc on major bump to 61

(From upstream SVN r38452)
---
 libmpcodecs/vd_ffmpeg.c | 4 ++--
 libmpcodecs/ve_lavc.c   | 2 +-
 2 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/libmpcodecs/vd_ffmpeg.c b/libmpcodecs/vd_ffmpeg.c
index c22397a..cf7eb27 100644
--- a/libmpcodecs/vd_ffmpeg.c
+++ b/libmpcodecs/vd_ffmpeg.c
@@ -515,8 +515,8 @@ static void uninit(sh_video_t *sh){
             mp_msg(MSGT_DECVIDEO, MSGL_INFO, "QP: %d, count: %d\n", i, ctx->qp_stat[i]);
         }
         mp_msg(MSGT_DECVIDEO, MSGL_INFO, MSGTR_MPCODECS_ArithmeticMeanOfQP,
-            ctx->qp_sum / avctx->frame_number,
-            1.0/(ctx->inv_qp_sum / avctx->frame_number)
+            ctx->qp_sum / avctx->frame_num,
+            1.0/(ctx->inv_qp_sum / avctx->frame_num)
             );
     }
 
diff --git a/libmpcodecs/ve_lavc.c b/libmpcodecs/ve_lavc.c
index b808be6..36ff6fd 100644
--- a/libmpcodecs/ve_lavc.c
+++ b/libmpcodecs/ve_lavc.c
@@ -741,7 +741,7 @@ static int encode_frame(struct vf_instance *vf, AVFrame *pic, double pts){
     int res;
 
     if(pts == MP_NOPTS_VALUE)
-        pts= lavc_venc_context->frame_number * av_q2d(lavc_venc_context->time_base);
+        pts= lavc_venc_context->frame_num * av_q2d(lavc_venc_context->time_base);
 
     if(pic){
 #if 0
-- 
2.46.0

